cmd/mattermost/commands/config.go

```diff
@@ -102,3 +117,120 @@ func configSubpathCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func configGetCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// create the model for config
+	// Note: app.Config() returns a pointer, make appropriate changes
+	config := app.Config()
+
+	// get the print config setting and any error if there is
+	out, err := printConfigValues(configToMap(*config), strings.Split(args[0], "."), args[0])
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("%s", out)
+
+	return nil
+}
+
+// printConfigValues function prints out the value of the configSettings working recursively or
+// gives an error if config setting is not in the file.
+func printConfigValues(configMap map[string]interface{}, configSetting []string, name string) (string, error) {
```

Comment:
Also, this doesn't block the PR or anything, but it might be nice to have this functionality available elsewhere. I could see something like a method on the Config type that just just returns the value as an `interface{}` and then the CLI command could just print that as a string. Something like
```go
func (config *Config) GetValueByPath(path string) (interface{}, error) {
    // ...
}
```

Reply:
I don't get you. What functionality do you mean? Can you explain it with an example, please?

Reply:
@hmhealey I have made the changes in the description of the command as you suggested, do I also have to move this command in some other file?

Reply:
No, this is totally fine as it is. It was just a suggestion for something that could be used in the future.